Docs: Update security log wording (#9811)
readthedocs/readthedocs.org: docs/user/security-log.rst

Security Log

> ============

Reviewer: I remember you we could change the title to something like: "Security log: audit organization's account".

Reviewer 2: Good idea to promote "auditing" to the title - how about "Security log and auditing"?

Author: Yea, I tried that, but I was worried it didn't match the other titles in our docs.

Security logs allow you to audit what has happened recently in your organization or account.
This feature is quite important for many security compliance programs,
as well as the general peace of mind of knowing what is happening on your account.
We store the IP address and the browser used on each event,

> so that you can confirm this access was from the intended person.

Reviewer 2: ```suggestion
so that you can confirm this access was from the intended person.

Security logs are only visible to organization owners.
You can invite other team members as owners.
```

I was wondering where to include this. It's a detail but it's also "general" so works best in the introduction.

> .. _User-Agent: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/User-Agent
> 

Reviewer: ```suggestion
```

We removed the reference.

To access your organization logs:

- Click on :guilabel:`<Username dropdown>`
- Click on :guilabel:`Organizations`
- Click on :guilabel:`<Organization name>`
- Click on :guilabel:`Settings`
- Click on :guilabel:`Security Log`

Your organization security log is a great place to check periodically to ensure there hasn't been unauthorized access to your organization.

If you have any additional information that you wished the security log was capturing,
you can always reach out to :doc:`/support`.

> .. seealso:: We have other security related information in our :doc:`/security` page.

Reviewer: ```suggestion
.. seealso:: 

    :doc:`/security`
      Security information related to our own platform, personal data treatment, how to report a security issue and more.
```

I think we have been using a style like this where we put the link in a definition list and write a small sentence about what it contains.

If you accept it, make sure it renders properly 😄

Reviewer 2: Yes! This is a great pattern, it makes it easy to list many different related articles instead of having a lot of "seealso" boxes.
